PowerPointLabs/PowerPointLabs/PictureSlidesLab/Views/PictureSlidesLabWindow.xaml.cs

```diff
@@ -532,7 +532,21 @@ private void ImageSelectionListBox_OnKeyDown(object sender, KeyEventArgs e)
         /// <param name="e"></param>
         private void Splitter_OnDragDelta(object sender, DragDeltaEventArgs e)
         {
-            StylesPreviewGrid.Width = StylesPreviewGrid.ActualWidth + e.HorizontalChange;
+            double newWidth = StylesPreviewGrid.ActualWidth + e.HorizontalChange;
```

Comment:
I think it might be better if you extract this into a Setter. This way, others will not unintentionally violate this rule.

Reply:
Are you suggesting extracting `StylesPreviewGrid.ActualWidth + e.HorizontalChange` into another method?

Reply:
@ChesterSng Actually I meant the code below, sorry. The <minWidth and >maxWidth part.

Reply:
Hi @YuPeiHenry, I considered your suggestion but considering that adjusting the splitter window is the only method that will adjust the width of `StylesPreviewGrid`, I think it is not necessary to create a separate setter for it. It is sufficient to contain the entire logic in this function.